### PR TITLE
Added foundation-rails

### PR DESCRIPTION
**Pull request by erniejamison**
The install overwrites app/views/layouts/application.html.erb. I can do a manual install if this is not ideal. Really it just added some embedded Ruby to grab the title and the responsive meta tag.

### CODE REVIEW

**Comment by erinmorrissey** [comment]
Kinda confused why a schema.rb file was created, but maybe that's not a problem? Looks good to me otherwise.

**Comment by erniejamison** [comment]
@erinmorrissey I ran db:create:all and migrate it was just sort of an auto-pilot thing/conditioned behavior. I can resubmit or we can just leave it.

File: Gemfile [hunk 1]
```diff
@@ -44,4 +44,7 @@ group :development, :test do
 end
 
 # Use Rails 12factor
-gem 'rails_12factor', group: :production
\ No newline at end of file
+gem 'rails_12factor', group: :production
+
+# Use foundation-rails
+gem 'foundation-rails'
```
Comment (lasley) [inline Gemfile]: All files should end with a newline. Crossed out circle icon indicates that is not the case

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by lasley** [comment]
Schema will be fine; we'll need it at some point anyways. One comment, non-blocking 👍

**Comment by lasley** [comment]
Welcome to conflict land, party of 1. Get used to this in the beginning unfortunately - the Rails gemlock is a freaking conflict nightmare.

After resolving the conflict and fixing my comment, drop a note saying you're done for a final review. Anything that was in conflict should have another review done before merge, even though we already have our two thumbs. This is because it is common to screw things up in conflict resolution 😉

**Comment by erniejamison** [comment]
Done and ready for final review. No actually not.

**Comment by lasley** [comment]
https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/

**Comment by erniejamison** [comment]
Ready for final review.

File: Gemfile.lock [hunk 2]
```diff
@@ -37,10 +37,14 @@ GEM
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
     arel (6.0.3)
+<<<<<<< HEAD
```
Comment (lasley) [inline Gemfile.lock]: Conflict zones are still present

**Comment by erniejamison** [comment]
No conflict zones ready for final review.

**Comment by lasley** [comment]
Hahah luck of the draw, conflict again! You're the next merge, promise 😛

**Comment by lasley** [comment]
Resolution looks good, merging

**Pull request merged**